### DISCUSSION TITLE
Getting errors in console

### DISCUSSION THREAD

**Discussion by hsanand61564** [iTwin/itwinjs-core#1797] [Q&A] [answered]
Hi

I am getting a lot of these errors in my console, can you please suggest something for the same.
I am using azure blog storage for tile cache
![image](https://user-images.githubusercontent.com/60655118/124616308-9964a200-de93-11eb-91d5-d9387e491c5c.png)

**Comment by hsanand61564** [comment, marked as answer]
Any Help on my issue?

**Reply by pmconne** [reply]
This is a browser problem, not an iModel.js problem. A quick web search will produce many suggestions for fixing it such as configuring browser, flushing DNS cache, etc.

**Reply by pmconne** [reply]
If problems persist please provide details. Closing for now.
